synth-publish.go

```diff
@@ -220,8 +221,41 @@ func checkPublishingStatus(latest postedData, result chan<- publicationResult, s
 
 	switch resp.StatusCode {
 	case http.StatusOK:
+		cmdR := exec.Command("kubectl", "delete", "cm", "synthetic-image-alarm", "--ignore-not-found")
+		cmdI := exec.Command("kubectl", "delete", "pods", "--selector=job-name=image-trace-job", "--ignore-not-found=true")
+
+		errR := cmdR.Run()
+		if errR != nil {
+			log.Fatalf("cmdR.Run() failed with %s\n", errR)
```

Comment:
Using `log.Fatalf` will make the program to exit, is this desired in this case?

Reply:
Now replaced with log.Printf